pkg/networkservice/chains/nsmgr/single_test.go

```diff
@@ -0,0 +1,262 @@
+// Copyright (c) 2020-2021 Doc.ai and/or its affiliates.
```

Comment:
Seems to me these changes are not related to the issue.

Reply:
Moved to another PR - https://github.com/networkservicemesh/sdk/pull/885.